libnmstate/state.py

```diff
@@ -76,6 +104,34 @@ def interfaces(self):
         """ Indexed interfaces state """
         return self._ifaces_state
 
+    @property
```

Comment:
After doing route addition/removal, the `self._routes` might not be sorted anymore.

The line `self.routes = []` require a setter.

Reply:
I do no follow, why does the possibility that it is not sorted require this to be a property? `self.routes = []` works fine if `self.routes` is a normal attribute/property, i.e. initialized in the constructor instead of using `self._routes` there...

Reply:
I just align with old codes style:
```python
        self._ifaces_state = State._index_interfaces_state_by_name(self._state)
```

If you still allows `self.routes = self._state.get(Route.KEY, {}).get(Route.CONFIG, [])`, let me know, I will change to it.